zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -750,6 +750,35 @@ function NotebookCtrl($scope, $route, $routeParams, $location, $rootScope,
 
   $scope.savePermissions = function() {
     convertPermissionsToArray();
+    if ($scope.isOwnerEmpty()) {
+      BootstrapDialog.show({
+        closable: false,
+        title: 'warning Permissions',
+        message: 'Please fill the [Owners] field. If not, it will set as current user.\n\n' +
+          'Current user : [ ' + $rootScope.ticket.principal + ']',
+        buttons: [
+          {
+            label: 'OK',
```

Comment:
instead of having an `OK` button to set/change permission, how about having it as `Set` to be more clear?